packages/core/core/src/BundleGraph.js

```diff
@@ -294,6 +300,61 @@ export default class BundleGraph {
       });
   }
 
+  getBundleGroupsReferencedByBundle(
```

Comment:
Kinda weird name since it actually returns both bundle groups and dependencies... but idk what else it could be